app/views/Settings/ImportFromUrl.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import {
+  StyleSheet,
+  View,
+  TextInput,
+  Alert,
+  NativeModules,
+} from 'react-native';
+import { useTranslation } from 'react-i18next';
+import importLocationsApi from '../../api/import/importLocationsApi';
+
+import { NavigationBarWrapper } from '../../components/NavigationBarWrapper';
+
+import { Typography } from '../../components/Typography';
+import { Button } from '../../components/Button';
+import { NavigationProp } from '../../navigation';
+
+import { Forms } from '../../styles';
+
+type ImportFromUrlProps = {
+  navigation: NavigationProp;
+};
+
+const ImportFromUrl = ({ navigation }: ImportFromUrlProps): JSX.Element => {
+  const { t } = useTranslation();
+  const [url, setUrl] = useState('');
+
+  // TODO: add logic to filter mocked data. We should agree on flag name
+
+  const importData = async () => {
+    try {
+      // Data must be an array of objects!
+      const mockData = await importLocationsApi(url);
+
+      if (mockData) {
+        await NativeModules.SecureStorageManager.importGoogleLocations(
+          mockData,
+        );
+        return navigation.goBack();
+      }
+      throw new Error();
```

Comment:
Can this error throw a message & be located in the API layer? I think we should also validate the data looks correct (is an array of the right shape) to prevent testing back and forth with figuring out issues

in api layer:
```js
if (!mockData) {
  throw new Error("No mock data found at <url>")
}
if (!validatePayload(mockData)) {
  throw new Error("Invalid mock data, expected <...>") 
}
```